### DISCUSSION TITLE
Missing "./react" export in "zustand" package

### DISCUSSION THREAD

**Discussion by jordanbaron** [pmndrs/zustand#1289] [Q&A]
Currently when I try to use any sort of Zustand state, even from the docs, I get an error saying `Missing "./react" export in "zustand" package`. I am using Vite and TypeScript.

```
const bears = useBearStore((state) => state.bears);

  useEffect(() => {
    console.log(bears);
  }, []);
```

Here is my vite.config.ts

```
import { defineConfig } from 'vite';
import react from '@vitejs/plugin-react';
import * as path from 'path';

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [react()],
  resolve: {
    alias: [{ find: '@', replacement: path.resolve(__dirname, 'src') }],
  },
});
``

**Comment by dai-shi** [comment]
We don't have `./react` export from the beginning.
Someone asked the same thing before. Does vite do something weird?

**Comment by dai-shi** [comment]
Does anyone else have similar experience?

**Reply by jordanbaron** [reply]
ahh i feel dumb I was manually importing `create` and VS Code had an autocomplete suggestion to import from `zustand/react;` and I didn't realize that was just importing the type file and since it's a type file there weren't any other errors. My bad :/

**Reply by dai-shi** [reply]
I see, so it's vscode that does wrong suggestion.. 😦
